Componets/Slider/slider.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Sanjay Jatti">
+    <meta name="description" content="Matrix - The perfect UI library for styling your website">
+    <title>Matrix | Slider</title>
+    <link rel="icon" type="image/png" href="/Assets/favicon_io/favicon.ico">
+    <link rel="stylesheet" href="/style.css">
+    <script src="https://kit.fontawesome.com/52c15eb80a.js" crossorigin="anonymous"></script>
+    <script src="/index.js"></script>
```

Comment:
add JavaScript file bottom of page . 
Note : When you have scripts loading in your header, they stop other downloads from taking place! This includes your styling, and will also stop your images from downloading until the script has finished.